### PR TITLE
Release/2.0.0

### PR DESCRIPTION
**Pull request by ikbalkaya**
Version 2.0.0

### CODE REVIEW

**Review by AndyNicks — APPROVED** [review]

**Review by stmoreau — APPROVED** [review]

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,33 @@
 # Changelog
 
+
+## [v2.0.0](https://github.com/ably/kafka-connect-ably/tree/v2.0.0) (2022-04-20)
+
+[Full Changelog](https://github.com/ably/kafka-connect-ably/compare/v2.0.0...HEAD)
```
Comment (QuintinWillison) [inline CHANGELOG.md:6]: This needs to work once the release is done... Looking at the previous release tag, I expect this should be:

```suggestion
[Full Changelog](https://github.com/ably/kafka-connect-ably/compare/v1.0.3...v2.0.0)
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,33 @@
...
+**Implemented enhancements:**
+
+- Optimize integration tests [\#58](https://github.com/ably/kafka-connect-ably/issues/58)
+- Remove client.use.binary.protocol configuration  [\#57](https://github.com/ably/kafka-connect-ably/issues/57)
+- Add configuration validators [\#53](https://github.com/ably/kafka-connect-ably/issues/53)
+- Make message name configurable and interpolable [\#46](https://github.com/ably/kafka-connect-ably/issues/46)
+- Code reorganization and refinement [\#63](https://github.com/ably/kafka-connect-ably/pull/63) ([ikbalkaya](https://github.com/ikbalkaya))
+- Add distributed mode configuration file [\#62](https://github.com/ably/kafka-connect-ably/pull/62) ([ikbalkaya](https://github.com/ikbalkaya))
+- Remove client.use.binary.protocol configuration [\#60](https://github.com/ably/kafka-connect-ably/pull/60) ([ikbalkaya](https://github.com/ikbalkaya))
+- Add .editorconfig for IntelliJ formatting support [\#54](https://github.com/ably/kafka-connect-ably/pull/54) ([ikbalkaya](https://github.com/ikbalkaya))
```
Comment (QuintinWillison) [inline CHANGELOG.md:17]: Changes like this don't benefit users of the plugin so we tend not to include them in changelogs. I would suggest you exclude entries like this and focus instead on customer value.

**Review by QuintinWillison — CHANGES_REQUESTED** [review]
It would be helpful to link to the upgrade guide from the root readme, similar to what we do [here in `ably-go`](https://github.com/ably/ably-go#breaking-api-changes-in-version-12x).

File: CHANGELOG.md (hunk 1 above)
Comment (ikbalkaya) [reply CHANGELOG.md]: 96fbd7d83b59bb617857f7067b9474d24ccf0f59

File: CHANGELOG.md (hunk 2 above)
Comment (ikbalkaya) [reply CHANGELOG.md]: b919c3fda69c9ece71804f600f95424509177a6c

Comment (ikbalkaya) [reply CHANGELOG.md]: Also removed a bit more entries here 1c9357f4417321fd208c53a7f525050a2fdff6d4 and tried to limit to entries which should be useful for customers

File: README.md [hunk 3]
```diff
@@ -114,6 +114,9 @@ An example cURL command to start the connector in distributed mode is:
     data: {"id":"1543960661:0:2","clientId":"kafka-connect-ably-example","connectionId":"SuJTceISnT","timestamp":1623496744538,"encoding":"base64", "channel":"kafka-connect-ably-example","data":"bWVzc2FnZSAz","name":"sink"}
     ```
 
+## Breaking API Changes in Version 2.0.0
+
+Please see our [Upgrade / Migration Guide](UPDATING.md) for notes on changes you need to make to your configuration to update it with changes introduced by version 2.0.0 of the connector.
```
Comment (QuintinWillison) [inline README.md:119]: Please always include vertical whitespace around headings... so you need an empty line below this para before the 'Configuration' H2.

**Review by QuintinWillison — APPROVED** [review]

File: README.md (hunk 3 above)
Comment (ikbalkaya) [reply README.md:119]: Good point :) bfb40948a5b9d670644ae23fc5c2a9910b123289

**Pull request merged**